test/e2e/service.go

```diff
@@ -38,6 +38,7 @@ var _ = Describe("Services", func() {
 	var c *client.Client
 	// Use these in tests.  They're unique for each test to prevent name collisions.
 	var namespace0, namespace1 string
+	var namespace *api.Namespace
```

Comment:
Since you are using only namespace.Name you can just remember its name not the whole namespace.